docs/06-concepts/04-exceptions.md

```diff
@@ -50,3 +50,35 @@ catch(e) {
   print('Something else went wrong.');
 }
 ```
+
+### Default Values in Exceptions
+
+Serverpod supports the use of default values for fields in exceptions, just like it does for models. This feature allows you to define default values for any supported field type directly within your exception definitions.
+
+#### How to Define Default Values
+
+To define a default value for a field in an exception, simply specify the `default` or `defaultModel` keywords, just as you would in a model.
+
+**Example:**
+
+```yaml
+exception: MyException
+fields:
+  message: String, default="An error occurred"
+  errorCode: int, default=1001
+```
+
+In the example above:
+
+- The `message` field will default to `"An error occurred"` if not provided.
+- The `errorCode` field will default to `1001`.
+
+#### Important Note
+
+Since exceptions are never persisted to the database, we do not support `defaultPersist` for exception fields. This means that only `default` and `defaultModel` are relevant in this context.
```

Comment:
```suggestion
Since exceptions are never persisted in the database, the `defaultPersist` keyword is not supported.
```

Reply:
Maybe even add it as a `::note` or `::info` depending on what looks best.

Reply:
i went with ::info, its better